src/xenia/ui/windowed_app_main_win.cc

```diff
@@ -351,6 +351,9 @@ int WINAPI wWinMain(HINSTANCE hinstance, HINSTANCE hinstance_prev,
   g_xenia_exe_base = reinterpret_cast<uintptr_t>(hinstance);
   g_xenia_exe_size = modinfo.SizeOfImage;
 
+  // Set "Above Normal" priority.
+  SetPriorityClass(GetCurrentProcess(), ABOVE_NORMAL_PRIORITY_CLASS);
```

Comment:
Still I would leave it to user to select priority class by some config option if possible. For example via some config option that allows to select:
0 - Normal
1 - Above Normal
2 - High

Problem is that I don't know if config is parsed at this point. I kinda doubt it, but it should be checked